core/src/test/java/org/elasticsearch/bwcompat/OldIndexBackwardsCompatibilityIT.java

```diff
@@ -120,7 +120,7 @@ public static void tearDownStatics() {
     public Settings nodeSettings(int ord) {
         return Settings.builder()
                 .put(MergePolicyConfig.INDEX_MERGE_ENABLED, false) // disable merging so no segments will be upgraded
-                .put(RecoverySettings.INDICES_RECOVERY_CONCURRENT_SMALL_FILE_STREAMS, 30) // increase recovery speed for small files
+                .put(RecoverySettings.INDICES_RECOVERY_CONCURRENT_SMALL_FILE_STREAMS_SETTING.getKey(), 30) // increase recovery speed for small files
```

Comment:
You know, if the builder had an overload to take a `Setting` and you don't rename them then this diff could be a lot smaller. Its not that important though.